code-review.md

```diff
@@ -0,0 +1,64 @@
+# Code Review Guidelines
+
+Code reviews are important, both to ensure code quality, and to learn from each
+other. To get the most out of it, code authors should voice their expectations
+of a review, and reviewers should state the performed actions and areas of
+focus.
+
+The necessary prerequisite for a *line-by-line* review is that code authors
+make sure that the changes they propose actually make sense and work at a basic
+level. *Corollary: It should never happen that neither the author nor the
+reviewer actually tests the fix.* It is still okay to request feedback on work
+in progress, but especially then it is important to be explicit about the depth
+of the desired review.
+
+__As code author (pre-review):__
+
+1. Make sure your proposed change does what you think it should do. This also
+   means that it conforms to the [code
+   style](https://github.com/secure-systems-lab/code-style-guidelines) etc. *Do
+   not waste the reviewer's time with things you can easily get right in the
+   first place!*
+
+1. Send a PR, that follows our [PR
+   guideline](commits.md#pull-request-guidelines-wip), and describe what you
+   think are the aspects that require the largest share of a reviewer's
+   attention:
+   > "Please do / look for / look at the following: ...."
+
+1. Feel free to suggest the review to someone, e.g. by using [GitHub's *review
```

Comment:
Perhaps remove "Feel free to" so that a review is always suggest to a reviewer and doesn't end up in limbo.